### PR TITLE
GUVNOR-2738: Guided Decision Table Editor: Highlight rows in table when V&V report item selected

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-2738

This PR adds highlighting of rows in a Decision Table based on the Analysis Report selection.

### CODE REVIEW

**Comment by manstis** [comment]
@Rikkola as discussed on IRC...

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java [hunk 1]
```diff
@@ -597,6 +605,26 @@ void onUpdatedLockStatusEvent( final @Observes UpdatedLockStatusEvent event ) {
         }
     }
 
+    void onIssueSelectedEvent( final @Observes IssueSelectedEvent event ) {
+        if ( event == null ) {
```
Comment (Rikkola) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java]: Can this be null or are you just being sure?

Comment (Rikkola) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java]: Either way, I'm ok that it is there.

Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java]: It probably cannot be null (if invoked by CDI).. I guess I got carried away with defensive programming (this time!)

**Pull request merged**